docs/resources/service_compute.md

```diff
@@ -150,11 +150,13 @@ Required:
 
 Optional:
 
+- `auto_loadbalance` (Boolean) Denotes if this Backend should be included in the pool of backends that requests are load balanced against. Default `false`
```

Comment:
👋🏻 

Just to clarify that 'compute' services should NOT have `auto_loadbalance`.

The following PR was merged recently to _remove_ `auto_loadbalance` from the `fastly_service_compute` resource, so best to rebase `main` branch and ensure those changes aren't undone:
https://github.com/fastly/terraform-provider-fastly/pull/657

This means the `TestResourceFastlyFlattenBackendCompute` test  (line 137) should NOT add the `"auto_loadbalance":      false,`.

Thanks.